_security-analytics/log-types-reference/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Supported log types
+has_children: yes
+nav_order: 16
+redirect_from:
+   - /security-analytics/sec-analytics-config/log-types/
+---
+
+# Supported log types
+
+Logs contain raw data about events that happen throughout a system and in its separate parts. This reference describes each standard log type supported by Security Analytics and their mapping information.
+
+## Log type categories
+
+As of OpenSearch 2.11, log types are grouped by the following categories to help select, filter, and search the log types:
+
+- **Access Management** includes [AD/LDAP]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/ad-ldap/) , [Apache Access]({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/apache-access/), and [Okta](({{site.url}}{{site.baseurl}}/security-analytics/log-types-reference/okta/).
```

Comment:
I think there's an extra "(" for Okta causing the hyperlink not to show up correctly.